roman/server.py

```diff
@@ -44,6 +44,18 @@ def stop(self):
         if self.use_sim:
             self.env.disconnect()
 
+    def get_visual_obs_server(self):
+        """
+        TODO: this only works for the simulated robot. Implement it for the real robot as well.
```

Comment:
This only works for the simulated robot *in proc*.

Reply:
Yes, updated the comment to say so.